crates/lox_core/src/bodies/barycenters.rs

```diff
@@ -1,81 +1,208 @@
-use crate::bodies::NaifId;
-
+// Auto-generated by `lox_gen`. Do not edit!
+use super::{NaifId, PointMass};
 pub struct SolarSystemBarycenter;
-
 impl NaifId for SolarSystemBarycenter {
     fn id() -> i32 {
-        0
+        0i32
     }
 }
-
 pub struct MercuryBarycenter;
-
 impl NaifId for MercuryBarycenter {
     fn id() -> i32 {
-        1
+        1i32
+    }
+}
+impl PointMass for MercuryBarycenter {
+    fn gravitational_parameter() -> f64 {
+        22031.868551400003f64
     }
 }
-
 pub struct VenusBarycenter;
-
 impl NaifId for VenusBarycenter {
     fn id() -> i32 {
-        2
+        2i32
+    }
+}
+impl PointMass for VenusBarycenter {
+    fn gravitational_parameter() -> f64 {
+        324858.592f64
     }
 }
-
 pub struct EarthBarycenter;
-
 impl NaifId for EarthBarycenter {
     fn id() -> i32 {
-        3
+        3i32
+    }
+}
+impl PointMass for EarthBarycenter {
+    fn gravitational_parameter() -> f64 {
+        403503.2356254802f64
```

Comment:
I would like the generated code to be checked into the repo so that it can be inspected by whomever is interested in it.
IIUC, we would also need to separately publish the code generator crate so that it can be added as build-time dependency. Since the code generator is probably going to run rather infrequently (only when we add new traits), I do not think that this is what we want.